scripts/test.py

```diff
@@ -69,7 +77,7 @@ def monte_carlo_sampling_scipy(N_FEATURES, SAMPLES, centered=False, strength=2,
     """
     from scipy.stats import qmc    
     
-    sampler = qmc.LatinHypercube(d=N_FEATURES, centered=centered, strength=strength, optimization=optimization, seed=seed)
+    sampler = qmc.LatinHypercube(d=N_FEATURES, centered=centered, strength=1, optimization=optimization, seed=seed)
```

Comment:
Has there been a particular reason for applying hard-coding for `strength`?

Reply:
No reason in particular. It was a mistake. Thanks for catching that.